.circleci/conditional_skip.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+SKIP_TAG="\[skip tests\]"
```

Comment:
Can't we also have the same behaviour with `[skip ci]`?

Reply:
No, both `[skip ci]` and `[ci skip]` are hardcoded and prevent the entire ci from running.

Reply:
Ah damn that sucks. So we should avoid [skip ci] from here and on then.